### ISSUE TITLE
Router with `/` in their name messes up with dashboard

### ISSUE THREAD

**Issue by sylr** [containous/traefik#5611]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [x] No

### What did you do?

I've clicked on following HTTP router

![Screenshot_84](https://user-images.githubusercontent.com/153052/66482513-65343580-eaa3-11e9-889d-b3aa16421fa7.png)

### What did you see instead?

![Screenshot_85](https://user-images.githubusercontent.com/153052/66482830-1935c080-eaa4-11e9-92f9-f80ecda9f014.png)

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
2.0.1
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
    [log]
      level = "INFO"
      format = "json"
    [accessLog]
      format = "json"
    [entryPoints.http]
      address = ":80"
    [entryPoints.http-internal]
      address = ":81"
    [entryPoints.https]
      address = ":443"
    [entryPoints.traefik]
      address = ":8080"
    [entryPoints.traefik-ext]
      address = ":8081"
    [providers.kubernetesIngress]
       ingressClass = "traefik"
    [providers.kubernetesCRD]
    [providers.file]
      directory  = "/config/static"
      watch = true
    [api]
      insecure = true
      dashboard = true
      debug = true
    [metrics.prometheus]
      entryPoint = "traefik"
      buckets = [0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8,0.9,1.0,1.5,2.0]
```

Ingress kubernetes:

```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: traefik
  creationTimestamp: "2019-07-31T14:42:45Z"
  generation: 2
  labels:
    app: pwet
    release: pwet
  name: pwet
  namespace: pwet
  resourceVersion: "90246831"
  selfLink: /apis/extensions/v1beta1/namespaces/pwet/ingresses/pwet
  uid: 7553afb4-b3a1-11e9-a70e-000d3a482661
spec:
  rules:
  - host: pwet.com
    http:
      paths:
      - backend:
          serviceName: pwet
          servicePort: 8080
        path: /
      - backend:
          serviceName: pwet
          servicePort: 8080
        path: /api/v1/
  tls:
  - hosts:
    - pwet.com
    secretName: pwet-tls
status:
  loadBalancer: {}
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
(paste your output here)
```

**Comment by sylr** [comment]
Related to: https://github.com/containous/traefik/pull/5612/files

Although there are other cases where a `/` can end up in the router name

**Comment by traefiker** [comment]
Closed by #5612.

**Comment by sylr** [comment]
Still happening in 2.0.2

![Screenshot_86](https://user-images.githubusercontent.com/153052/66558338-adab2c00-eb53-11e9-87f8-c301095568c0.png)

@dduportal could you re-open please ?

**Comment by dduportal** [comment]
@sylr can you provide a reproduction case please?

**Comment by dduportal** [comment]
Ref #5623

**Comment by traefiker** [comment]
Closed by #5623.
